tests/e2e/operationstorm.go

```diff
@@ -208,4 +211,115 @@ var _ = utils.SIGDescribe("[csi-block-vanilla] Volume Operations Storm", func()
+		3. Wait for PV to be provisioned.
+		4. Wait for PVC's status to become Bound.
+		5. Create pods using PVCs on specific node
+		6. Wait for Disk to be attached to the node.
+		7. Delete Namespace  and Wait for volumes to be deleted and Volume Disk to be detached from the Node.
+	*/
+
+	ginkgo.It("[csi-block-vanilla] [csi-file-vanilla] [csi-guest] [csi-supervisor] Delete namespace to confirm all volumes and pods are deleted", func() {
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+		ginkgo.By(fmt.Sprintf("Running test with VOLUME_OPS_SCALE: %v", volumeOpsScale))
+		ginkgo.By("Creating Storage Class")
+
+		// decide which test setup is available to run
+		if vanillaCluster {
+			ginkgo.By("CNS_TEST: Running for vanilla k8s setup")
+			scParameters = nil
+			storagePolicyName = ""
+		} else if supervisorCluster {
+			ginkgo.By("CNS_TEST: Running for WCP setup")
+			profileID := e2eVSphere.GetSpbmPolicyID(storagePolicyName)
+			scParameters[scParamStoragePolicyID] = profileID
+			// create resource quota
+			createResourceQuota(client, namespace, "100Gi", storagePolicyName)
+		} else if guestCluster {
+			ginkgo.By("CNS_TEST: Running for GC setup")
+			scParameters[svStorageClassName] = storagePolicyName
+			svcClient, svNamespace := getSvcClientAndNamespace()
+			setResourceQuota(svcClient, svNamespace, rqLimit)
+		}
+		if guestCluster {
+			storageclass, err = createStorageClass(client, scParameters, nil, "", "", true, "")
+		} else {
+			scSpec := getVSphereStorageClassSpec(storagePolicyName, scParameters, nil, "", "", false)
+			storageclass, err = client.StorageV1().StorageClasses().Create(ctx, scSpec, metav1.CreateOptions{})
+		}
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		defer func() {
+			err := client.StorageV1().StorageClasses().Delete(ctx, storageclass.Name, *metav1.NewDeleteOptions(0))
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		}()
+
+		count := 0
+		for count < volumeOpsScale {
+			ginkgo.By("Creating PVCs using the Storage Class")
+			pvclaims[count], err = fpv.CreatePVC(client, namespace, getPersistentVolumeClaimSpecWithStorageClass(namespace, diskSize, storageclass, nil, ""))
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			count++
+		}
+
+		ginkgo.By("Waiting for all claims to be in bound state")
+		persistentvolumes, err = fpv.WaitForPVClaimBoundPhase(client, pvclaims, framework.ClaimProvisionTimeout)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		podCount := 0
+		for podCount < volumeOpsScale {
+			ginkgo.By("Creating pod to attach PVs to the node")
+			pvclaim = pvclaims[podCount]
+			podArray[podCount], err = createPod(client, namespace, nil, []*v1.PersistentVolumeClaim{pvclaim}, false, "")
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			podCount++
+		}
+
+		defer func() {
+			err = client.CoreV1().Namespaces().Delete(ctx, namespace, *metav1.NewDeleteOptions(0))
+			if !apierrors.IsNotFound(err) {
```

Comment:
why are we ignoring `IsNotFound`?

Reply:
The same logic was present on line 161 - Replicated the same for this test also